tern/__main__.py

```diff
@@ -77,6 +80,7 @@ def do_main(args):
 
 def main():
     parser = argparse.ArgumentParser(
+        formatter_class=argparse.RawTextHelpFormatter,
```

Comment:
So far I've seen that the help text wraps at a reasonable length. I'm not sure what adding this does.

Reply:
That was not my experience when I was testing these changes but perhaps it was just my environment? Do you want to give it a try with and without this and see if you have the same experience?

Reply:
Yup I'll test it out. It's probably the right call if the version text doesn't wrap like the rest of them.

Reply:
Sorry, I misinterpreted. You want the new line in the formatting. Looks good!